commands/project/listBuilds.ts

```diff
@@ -29,22 +26,23 @@ const { isHubSpotHttpError } = require('@hubspot/local-dev-lib/errors/index');
 
 const i18nKey = 'commands.project.subcommands.listBuilds';
 
-exports.command = 'list-builds [path]';
+exports.command = 'list-builds';
 exports.describe = uiBetaTag(i18n(`${i18nKey}.describe`), false);
 
 exports.handler = async options => {
   await loadAndValidateOptions(options);
 
-  const { path: projectPath, limit, derivedAccountId } = options;
+  const { project: projectFlagValue, limit, derivedAccountId } = options;
 
   trackCommandUsage('project-list-builds', null, derivedAccountId);
 
-  const cwd = projectPath ? path.resolve(getCwd(), projectPath) : getCwd();
-  const { projectConfig, projectDir } = await getProjectConfig(cwd);
-
-  validateProjectConfig(projectConfig, projectDir);
+  let projectName = projectFlagValue;
 
-  logger.debug(`Fetching builds for project at path: ${projectPath}`);
+  if (!projectName) {
+    const { projectConfig, projectDir } = await getProjectConfig();
```

Comment:
Does the user need to run this command from within a project folder? What happens if `getProjectConfig` fails?

Reply:
So this command can be run from both inside and outside of a project. If the dev runs it from outside of a project they need to include the `[--project]` flag. If they don't include that flag, we'll throw an error and tell them that they need to run it from inside a project. If they run from inside a project it'll just default to the project they're in.

The ideal way to run this is from inside a project, but we do support running it with `--project` to target a specific project in your hubspot account